README.md

```diff
@@ -1,3 +1,11 @@
+Found this neat little node called seqNum which basically acts as a counter - I haven't used it yet. Why haven't I used it yet...
+
+        https://github.com/srijs/node-seqnum
```

Comment:
ooo interesting! 🤔

Reply:
Shame I forgot about it and didn't use it!